documentation/src/components/home.css

```diff
@@ -26,20 +26,47 @@ governing permissions and limitations under the License.
 #features {
     display: flex;
     flex-direction: row;
+    flex-wrap: wrap;
```

Comment:
The following changes allow us to delivery in the smallest case one single column of three rows and in the largest one single row of three columns, as well as support awkward in the middle sizes but delivering two columns, one with a single element and the second with two.